composer.json

```
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.0",
-        "sylius/sylius": ">=1.11 <1.14"
+        "sylius/sylius": "^2.0.0-alpha.2"
```

This should be enough and cover you until the stable release is done. As Sylius v2.0.0-ALPHA-3 was just released and it is planned to go up to ALPHA-5 before the BETA releases start. You will not need to to upgrade all these packages with every alpha there is.
 
```suggestion
        "sylius/sylius": "^2.0.0@alpha"
```